src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -69,7 +62,8 @@
           sm="9"
         >
           <p>
-            Upload documents that support proof of authorization from your home jursidiction.
+            Upload one or more documents that show proof of authorization to continue out of your
+            previous jursidiction.
```

Comment:
From here down is the new design, ie:

![image](https://github.com/user-attachments/assets/f1636504-7b98-4806-bacc-37d7c60c4884)